### DISCUSSION TITLE
ChatFilter method getId

### DISCUSSION THREAD

**Discussion by malloc32** [tdlight-team/tdlight-java#72] [Q&A] [answered]
I need to use the class ChatFilter, but in Telegram API this class has a public method get_id(), but I do not found this method in this library... the id of a chatFilter is very necessary. I am doing something wrong?
Thank you.

**Comment by cavallium** [comment, marked as answer]
Telegram API is not TDLib API.
You should look here for the TDLib API documentation: https://tdlight-team.github.io/tdlight-docs/tdlight.api/it/tdlight/jni/package-tree.html

**Comment by malloc32** [comment]
But without this method chatfilter can not be used...
Here you can see the method, it is in TDLib.
https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1chat_filter.html

Thank you.

**Comment by cavallium** [comment]
get_id is not related to ChatFilter, it's an internal tdlib method used by the c++ api to determine the type of each tdlib object.

**Comment by malloc32** [comment]
Then I have a problem to LoadChats using a ChatFilterList using a ChatFilter. Ummm Ok, I need use CreateChatFilter.
Thank you and sorry.

**Comment by cavallium** [comment]
Can you show me your code and explain me what are you trying to achieve?

**Reply by malloc32** [reply]
Thank you very much, the problem was that I'm newbie.... finally my code was like this:
`ChatFilter chatFilter = new TdApi.ChatFilter("test", null, null, null, new long[] { chatId }, true, true, true,
                false, false, false, true, true);`
`client.send(new TdApi.CreateChatFilter(chatFilter), defaultHandler);`
It works.
Thank you very much and sorry.
